examples/diagnostic_protocol/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.16)
-
+project(diagnostic_protocol_example)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+find_package(isobus CONFIG)
```

Comment:
We don't set this to REQUIRED for building examples from top level. Regardless of whether examples are built from top level or not, the linkage below will fail if the targets are not found since we use alias libs. This is desireable behavior. If you decide you want to build examples from top level by default, we may want to add the QUIET keyword here to silence the warnings about the call to find_package not being successful.

Reply:
Added information about this in 77dd2cb1fc69bb01adc706298d82ac2774b0dfc6